htdocs/core/class/hookmanager.class.php

```diff
@@ -330,17 +330,17 @@ public function executeHooks($method, $parameters = array(), &$object = null, &$
 
 						if (isset($actionclassinstance->results) && is_array($actionclassinstance->results)) {
 							if ($resactiontmp > 0) {
-								$localResArray = $actionclassinstance->results;
-							} else {
 								$localResArray = array_merge_recursive($localResArray, $actionclassinstance->results);
+							} else {
```

Comment:
I think it should be the oposite: the merge when result is 0, and other value erased and not merged when result is > 0

Reply:
You're right. Thank you, I will close the PR.